velox/docs/develop/unsaferow.rst

```diff
@@ -20,17 +20,18 @@ The 'null bits' section contains one bit per column. 0 bit indicates a
 non-null value. 1 bit indicates a null value.
 
 The 'fixed-width data' section contains 8 bytes per column. Values of the
-fixed-width columns (booleans, integers, floating point numbers) are stored
-directly. These values must fit within 8 bytes. Long decimal columns are not
-supported.
+fixed-width columns (booleans, integers, floating point numbers, short decimals) are stored
+directly. These values must fit within 8 bytes.
 
-Values of the variable-width columns (strings, arrays, maps) are split between
+Values of the variable-width columns (long decimals, strings, arrays, maps) are split between
 fixed-width and variable-width sections. 8 bytes of the fixed-width section
 store the size and location of the value in the variable-width section.
 
 The 'variable-width data' stores values of the variable-width columns. Each
 value is aligned to 8 bytes.
 
+Long decimals are stored as binary, the content is serialized java BigInteger.
```

Comment:
Would it be possible to explain what does "serialized java BigInteger" looks like?